README.md

```diff
@@ -40,6 +40,16 @@ Add Orbit package to your project by adding the package dependency:
 
 ## Usage
 
+0. (Optional) Set custom fonts. [Circular Pro is only licensed for Kiwi.com domain, for non-Kiwi.com projects use an alternative](https://orbit.kiwi/foundation/typography/open-source/).
```

Comment:
"Set Circular Pro or custom fonts. [Circular Pro is licenced font...](...)"
...
"If you omit this steps, a default SF Pro font will be used."